pkg/sql/pg_catalog_test.go

```diff
@@ -170,9 +215,396 @@ func rewriteDiffs(t *testing.T, diffs PGCatalogTables, diffsFile string) {
 	}
 }
 
+// fixConstants helps to update catconstants that are needed for pgCatalog
```

Comment:
fixConstants updates catconstants that are needed for pgCatalog

Reply:
Done.